neard/Cargo.toml

```diff
@@ -116,6 +116,7 @@ nightly_protocol = [
   "nearcore/nightly_protocol",
 ]
 statelessnet_protocol = [
+  "nearcore/statelessnet_protocol",
```

Comment:
Is having this compile-time feature temporary? I feel like we should definitely look at making this a runtime flag if possible to avoid a matrix of 3 distinct neards if possible. It already is hard as-is to keep up with two configurations during regular development.

Reply:
Yeah, it will be removed after StatelessNet is complete (2 months?).
This protocol feature is needed to enforce specific protocol version everywhere, including tests. I doubt that passing protocol version to all involved structures is more practical than just have `PROTOCOL_VERSION` checks.